app/Models/Photo.php

```diff
@@ -284,7 +285,7 @@ protected function getFocalAttribute(?string $focal): ?string
 			return null;
 		}
 		// We need to format the framerate (stored as focal) -> max 2 decimal digits
-		return $this->isVideo() ? (string) round($focal, 2) : $focal;
+		return $this->isVideo() ? (string) round(intval($focal), 2) : $focal;
```

Comment:
Erm, I know it's late and I should be in bed already, but this makes no sense to me?!

Reply:
hummm I think I should have gone for float indeed.